tests/sentry/plugins/mail/activity/test_release.py

```diff
@@ -300,3 +300,67 @@ def test_no_committers(self):
         sent_email_addresses = {msg.to[0] for msg in mail.outbox}
 
         assert sent_email_addresses == {self.user3.email}
+
+    def test_uses_default(self):
+        user6 = self.create_user()
+        print user6.id
```

Comment:
remove print

Reply:
damn need to figure out why this isn't being caught in the pre commit linting